installer/deb/create_deb.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# This script generates a .deb file for the installation of MassaStation on a Debian-based Linux distribution.
+
+BUILD_DIR=builddeb
+PKGVERSION=0.0.0-dev
+
+DEB_NAME=massastation_$PKGVERSION\_amd64.deb
+SERVER_BINARY_NAME=massastation-server
+APP_BINARY_NAME=massastation-app
+
+# Print error message to stderr and exit with code 1.
+fatal() {
+    echo "FATAL: $1"
+    exit 1
+}
+
+# Download the latest release of MassaStation app.
+download_massastation_app() {
+    curl -L https://github.com/massalabs/Thyra-Menu-Bar-App/releases/latest/download/ThyraApp_linux-amd64 -o $APP_BINARY_NAME || fatal "failed to download $APP_BINARY_NAME"
+    chmod +x $APP_BINARY_NAME || fatal "failed to chmod $APP_BINARY_NAME"
+}
+
+# Download the latest release of MassaStation server.
+download_massastation_server() {
```

Comment:
Now that you are in the repo, you shouldn't download the last version but use the built binary instead.

Reply:
I was planning to do it with the CI. 
I did it now following you comment and will update the other scripts to do the same.